### ISSUE TITLE
RuboCop is not run

### ISSUE THREAD

**Issue by paulmenzel** [brigade/overcommit#331]
Using Overcommit version 0.31.0 and RuboCop 0.36.0, for some strange reason, RuboCop is not run.

```yaml
$ more .overcommit.yml
# Use this file to configure the Overcommit hooks you wish to use. This will
# extend the default configuration defined in:
# https://github.com/brigade/overcommit/blob/master/config/default.yml
#
# At the topmost level of this YAML file is a key representing type of hook
# being run (e.g. pre-commit, commit-msg, etc.). Within each type you can
# customize each hook, such as whether to only run it on certain files (via
# `include`), whether to only display output if it fails (via `quiet`), etc.
#
# For a complete list of hooks, see:
# https://github.com/brigade/overcommit/tree/master/lib/overcommit/hook
#
# For a complete list of options that you can use to customize hooks, see:
# https://github.com/brigade/overcommit#configuration
#
# Uncomment the following lines to make the configuration take effect.

PreCommit:
  Rubocop:
    enabled: true
    on_warn: fail # Treat all warnings as failures

  TrailingWhitespace:
    enabled: true
    exclude:
      - '**/db/structure.sql' # Ignore trailing whitespace in generated files
```

Running `git commit -a` just runs the whitespace check though.

```
$ git commit -a # left empty
Running pre-commit hooks
Checking for trailing whitespace.................[TrailingWhitespace] OK

✓ All pre-commit hooks passed

Running commit-msg hooks
Checking subject capitalization..................[CapitalizedSubject] OK
Checking for empty commit message......................[EmptyMessage] FAILED
Commit message should not be empty
Checking subject line.............................[SingleLineSubject] OK
Checking text width.......................................[TextWidth] OK
Checking for trailing periods in subject.............[TrailingPeriod] OK

✗ One or more commit-msg hooks failed
```

Running `rubocop` directly works.

```
$ rubocop app/models/booking.rb | grep Line
app/models/something.rb:12:121: C: Line is too long. [307/120]
[…]
```

RVM is used, but the Gem *rubocop* is in every environment I tested with.

**Comment by lencioni** [comment]
Does it work if you change `Rubocop:` to `RuboCop:` in your .overcommit.yml file?

**Comment by lencioni** [comment]
This seems like the same issue as #329. We have an [open issue](https://github.com/brigade/overcommit/issues/325) to have Overcommit display a warning of some kind when you've misspelled or otherwise incorrectly configured a hook.

Hope that helps!

**Comment by paulmenzel** [comment]
Thank you for the quick response! That was indeed it! Looks like I was bitten by a problem fixed in May by commit 4eeb0c42 (Change remaining "Rubocop"s to "RuboCop").
